packages/theme-chalk/src/message-box.scss

```diff
@@ -244,15 +244,3 @@
     opacity: 1;
   }
 }
-
-@keyframes msgbox-fade-out {
-  0% {
-    transform: translate3d(0, 0, 0);
-    opacity: 1;
-  }
-
-  100% {
-    transform: translate3d(0, -20px, 0);
-    opacity: 0;
-  }
-}
```

Comment:
Perhaps it should not be removed, but applied in line 232.

Reply:
`animation: msgbox-fade-in getCssVar('transition-duration') reverse;`  

But use reverse,  have less code

Reply:
You are right!